### PR TITLE
Added docs for using client-side ssl certificates

### PR DESCRIPTION
**Pull request by hemanth7787**
Closes https://github.com/encode/httpx/issues/1224

### CODE REVIEW

**Review by tomchristie — APPROVED** [review]
Fantastic, thanks!

Anyone else want to chime in with a review on this or are we good to go?

File: docs/advanced.md [hunk 1]
```diff
@@ -945,6 +945,32 @@ client = httpx.Client(verify=False)
 
 The `client.get(...)` method and other request methods *do not* support changing the SSL settings on a per-request basis. If you need different SSL settings in different cases you should use more that one client instance, with different settings on each. Each client will then be using an isolated connection pool with a specific fixed SSL configuration on all connections within that pool.
 
+### Client Side Certificates
+
+You can also specify a local cert to use as client side certificate, Either a path to an SSL certificate file, or   two-tuple of (certificate file, key file), or a three-tuple of (certificate file, key file, password)
```
Comment (j178) [inline docs/advanced.md:950]:
```suggestion
You can also specify a local cert to use as a client-side certificate, either a path to an SSL certificate file, or two-tuple of (certificate file, key file), or a three-tuple of (certificate file, key file, password)
```

File: docs/advanced.md [hunk 2]
```diff
@@ -945,6 +945,32 @@ client = httpx.Client(verify=False)
...
+Alternatively,
+
+```pycon
+>>> cert = ("path/to/client.pem", "path/to/client.key")
+>>> httpx.get("https://example.org", cert=cert)
+<Response [200 OK]>
+```
+
+or
+```
```
Comment (j178) [inline docs/advanced.md:968]:
```suggestion
```pycon
```

File: docs/advanced.md [hunk 3]
```diff
@@ -945,6 +945,32 @@ client = httpx.Client(verify=False)
 
 The `client.get(...)` method and other request methods *do not* support changing the SSL settings on a per-request basis. If you need different SSL settings in different cases you should use more that one client instance, with different settings on each. Each client will then be using an isolated connection pool with a specific fixed SSL configuration on all connections within that pool.
 
+### Client Side Certificates
+
+You can also specify a local cert to use as a client-side certificate, either a path to an SSL certificate file, or two-tuple of (certificate file, key file), or a three-tuple of (certificate file, key file, password)
+
+
```
Comment (tomchristie) [inline docs/advanced.md:952]:
```suggestion

```

Drop one newline here.

File: docs/advanced.md [hunk 4]
```diff
@@ -945,6 +945,32 @@ client = httpx.Client(verify=False)
...
+Alternatively,
+
+```pycon
+>>> cert = ("path/to/client.pem", "path/to/client.key")
+>>> httpx.get("https://example.org", cert=cert)
+<Response [200 OK]>
+```
+
+or
+```pycon
```
Comment (tomchristie) [inline docs/advanced.md:968]:
```suggestion
or

```pycon
```

Add a newline here.

File: docs/advanced.md [hunk 5]
```diff
@@ -945,6 +945,31 @@ client = httpx.Client(verify=False)
 
 The `client.get(...)` method and other request methods *do not* support changing the SSL settings on a per-request basis. If you need different SSL settings in different cases you should use more that one client instance, with different settings on each. Each client will then be using an isolated connection pool with a specific fixed SSL configuration on all connections within that pool.
 
+### Client Side Certificates
+
+You can also specify a local cert to use as a client-side certificate, either a path to an SSL certificate file, or two-tuple of (certificate file, key file), or a three-tuple of (certificate file, key file, password)
+```python
+import httpx
```
Comment (tomchristie) [inline docs/advanced.md:952]:
```suggestion
You can also specify a local cert to use as a client-side certificate, either a path to an SSL certificate file, or two-tuple of (certificate file, key file), or a three-tuple of (certificate file, key file, password)

```python
import httpx
```

**Comment by tomchristie** [comment]
Thanks @j178 - I've made a few whitespacing fixes too. Happy to go with this if you are?

**Comment by j178** [comment]
LGTM 💯

**Pull request merged**